src/side-navigation/side-navigation.scss

```diff
@@ -141,6 +141,7 @@
       &:focus:not(:focus-visible) {
         @include themed {
           box-shadow: inset $iui-xxs 0 0 0 t(iui-icons-color-primary);
+          color: t(iui-color-foreground-primary);
```

Comment:
This is still not completely fixing the problem. If you try selecting a side nav button, it has a leftover top border now.
![image](https://user-images.githubusercontent.com/9084735/144891310-14eee8aa-e67a-4f4b-be67-bc939b9b4972.png)

Maybe you won't need any of these overrides once you add `:where`.

Reply:
Yes, it's not needed when added `:where`. Removed.